docs/docs/en/guides/features/insights.md

```diff
@@ -81,17 +81,40 @@ To quickly access the dashboard, run `tuist project show --web` from the CLI.
 
 ![Dashboard with build insights](/images/guides/features/insights/builds-dashboard.png)
 
+## Tests {#tests}
+
+In addition to tracking build times, you can also monitor your tests. Test insights help you identify slow tests or quickly understand failed CI runs.
```

Comment:
This reads very time-oriented, but you can also track other things (errors, warnings, result). I'd rephrase it to be more generic.